iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/vector/HiveVectorizedReader.java

```diff
@@ -60,7 +61,8 @@ private HiveVectorizedReader() {
 
   public static <D> CloseableIterable<D> reader(InputFile inputFile, FileScanTask task, Map<Integer, ?> idToConstant,
       TaskAttemptContext context) {
-    JobConf job = (JobConf) context.getConfiguration();
+    // Tweaks on jobConf here are relevant for this task only, so we need to copy it first as context's conf is reused..
```

Comment:
Is this `not relevant`, or `should not change`?

Reply:
Hm what I meant is that any changes we make to the jobConf instance in the code below, should not affect anyone else. And for that reason I need to take a copy of it and do the adjustments on the copied instance.